cfpb/consumerfinance.gov
# Update timely response percentage

The percent of complaints that receive timely responses has gone up to 98%.

## Changes

- Bumped the percent of timely responses shown on `/data-research/consumer-complaints/` to 98%
- Fixed the missing bottom margin on the timely response callout at small screen sizes

## How to test this PR

1. Go to `/data-research/consumer-complaints/`
2. Confirm the number in the "complaints sent to companies get timely responses" callout is "98%" at all screen sizes (the content appears in the markup twice, once for large screens and once for small screens 😢 )
3. Confirm that callout has a 15px bottom margin at small screen sizes

## Screenshots

Large screens | Small screens
---- | ----
![large-screens](https://user-images.githubusercontent.com/1862695/110039945-044c7380-7d10-11eb-8d00-f7cc3d36621f.png) | ![small-screens](https://user-images.githubusercontent.com/1862695/110039957-08789100-7d10-11eb-980c-28ee9181c380.png)

## Checklist



  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
